pkgs/top-level/all-packages.nix

```diff
@@ -41686,6 +41686,8 @@ with pkgs;
 
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
+  ssg = with python3Packages; toPythonApplication ssg;
```

Comment:
The name `ssg` is short and may cause conflicts in the future.
Would you still add the attribute at the top level? Of course, if it is necessary, we should add it.

Reply:
> The name `ssg` is short and may cause conflicts in the future. Would you still add the attribute at the top level? Of course, if it is necessary, we should add it.

It's not really necessary, I'll remove it from the top level